public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -130,6 +131,7 @@ export class PanelModel {
     this.ensureQueryIds();
 
     this.restoreInfintyForThresholds();
+    this.updateOptions = this.updateOptions.bind(this);
```

Comment:
bit sure I like this bind, due use from outside. this should be handled in PanelChrome instead it  is the component that sets the onOptionsChange